go-controller/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ovn-org/ovn-kubernetes/go-controller
 go 1.13
 
 require (
-	github.com/Mellanox/sriovnet v1.0.1
+	github.com/Mellanox/sriovnet v1.0.2-0.20210112115703-ecc40df73c7c
```

Comment:
I released github.com/Mellanox/sriovnet v1.0.2

Reply:
Will update to the released version, thanks!